packages/pds/src/services/account/index.ts

```diff
@@ -100,6 +100,7 @@ export class AccountService {
         did,
         passwordScrypt,
         createdAt: new Date().toISOString(),
+        invitesDisabled: 0,
```

Comment:
This is 💯 as-is, but just for sharing knowledge's sake (and good chance you already know this)— since there's a default on the `invitesDisabled` column, you can mark it in the model as `Generated<0 | 1>`, meaning if it's not set the db will generate it.  That allows you to omit the column from places such as this.